rc/core/sh.kak

```diff
@@ -33,10 +33,10 @@ addhl -group /sh/comment fill comment
 }
 
 addhl -group /sh/code regex [\[\]\(\)&|]{1,2} 0:operator
-addhl -group /sh/code regex (\w+)= 1:identifier
+addhl -group /sh/code regex (\w+)= 1:attribute
 addhl -group /sh/code regex ^\h*(\w+)\h*\(\) 1:identifier
 
-addhl -group /sh/code regex \$(\w+|\{.+?\}|#|@|\?|\$|!|-|\*) 0:identifier
+addhl -group /sh/code regex \$(\w+|\{.+?\}|#|@|\?|\$|!|-|\*) 0:value
```

Comment:
You could make the argument that expansions are values, especially subshells. I want shell scripts to be highlighted with more than two colors (string/identifier), I think this is a valid change.